### DISCUSSION TITLE
chakra themes not available in `window.getComputedStyle` when running tests?

### DISCUSSION THREAD

**Discussion by georgienwankwo** [chakra-ui/chakra-ui#6302] [Q&A]
So I want to test for background changes in my project but I can't seem to figure out if I'm doing something wrong.

I use jest + React testing library for my unit tests. 

my dependencies are: 
`@chakra-ui/react`: "^2.1.2",
`react`: "^18.1.0",
`react-dom`: "^18.1.0"
`jest`: "^28.1.2",
`next`: "^12.1.6",
`jest-environment-jsdom`: "^28.1.2"

Here's my code:
`import { render } from "@testing-library/react";
import { Box } from "@chakra-ui/react";

const App = () => {
  return (
    <Box width="200px" height="200px" background="purple.400" data-testid="bg" />
  );
};

describe("Background Check", () => {
  it("has a purple background", () => {
    render(
      <ChakraProvider>
        <App />
      </ChakraProvider>
    );
    const background = document.querySelector("[data-testid=bg]");
    const computedStyle = window.getComputedStyle(background);
    console.log(computedStyle);
  });
});`

the computedStyle doesn't contain the background color which is from the chakra theme but does contain the width and height.

I would appreciate it if anyone can help. Thanks.

**Comment by lsfgrd** [comment]
I was having the same issue of not being able to see Chakra UI styles when using the responsive object syntax.

This was solved by wrapping my component under test with `ChakraProvider` in a custom render function.

This is basic config for the library but I forgot about it.
